staging/src/k8s.io/pod-security-admission/test/fixtures_procMount.go

```diff
@@ -0,0 +1,66 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package test
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/pod-security-admission/api"
+)
+
+func init() {
+	fixtureData_1_0 := fixtureGenerator{
+		expectErrorSubstring: "forbidden proc mount type",
+		generatePass: func(p *v1.Pod) []*v1.Pod {
+			p = ensureSecurityContext(p)
+			return []*corev1.Pod{
+				// set proc mount of container to a valid value
+				tweak(p, func(copy *v1.Pod) {
+					validProcMountType := v1.DefaultProcMount
+					copy.Spec.Containers[0].SecurityContext.ProcMount = &validProcMountType
+				}),
```

Comment:
combine the container and initContainer valid case to minimize number of fixtures while retaining coverage